Formula/arangodb.rb

```diff
@@ -28,8 +28,11 @@ class Arangodb < Formula
       :revision => "bbe29730e70dba609b57c469e8f863f032fabf3e"
   end
 
+  patch :DATA
+
   def install
-    ENV.cxx11
+    ENV.cxx
```

Comment:
`ENV.cxx` is the name of the C++ compiler. This line should simply be removed.

Reply:
line removed